### PR TITLE
Add ZTE, Supermicro vendors with  ZXA10 F8x2 GPON MTU series, Cisco Meraki AP MRxx series,  Supermicro model SYS-1019P-WTR

### PR DESCRIPTION
**Pull request by vittore**

### CODE REVIEW

**Comment by vittore** [comment]
[Add generic SC-24-double-port Fiber Patch Panel](https://github.com/netbox-community/devicetype-library/pull/843/commits/375c23ad59540a9bcbf4ec6da1d20e61a7f79582) Add generic fiber patch panel SC 24 double ports

**Comment by vittore** [comment]
Added:
- ZTE vendor  with  ZXA10 F8x2 GPON MTU series 
- Meraki AP MRxx series

**Comment by ryanmerolle** [comment]
This is a lot to review in 1 PR. I would recommend in the future to separate out your PRs by vendor.

**Comment by ryanmerolle** [comment]
1. Please do not delete the pre-commit config.  You do not have to use it if you do not want to.  Please add it back.
2. Review the test failures and address them.

**Comment by vittore** [comment]
> 1. Please do not delete the pre-commit config.  You do not have to use it if you do not want to.  Please add it back.
> 2. Review the test failures and address them.

Revision done.

File: device-types/Supermicro/SYS-1019P-WTR.yaml [hunk 1]
```diff
@@ -0,0 +1,71 @@
...
+    description: USB 2.0
+  - name: Header USB11
+    type: usb-a
+    description: USB 2.0
+  - name: Header USB12
+    type: usb-a
+    description: USB 2.0
+  - name: Header USB13
+    type: usb-a
+    description: USB 2.0
```
Comment (DanSheps) [inline device-types/Supermicro/SYS-1019P-WTR.yaml:57]: Invalid use of console server ports

File: device-types/Supermicro/SYS-1019P-WTR.yaml [hunk 2]
```diff
@@ -0,0 +1,71 @@
...
+console-ports:
+  - name: COM1
+    type: de-9
+    label: Rear
+
+  - name: COM2
+    type: de-9
+    description: Header
+  - name: Front VGA
+    type: other
```
Comment (DanSheps) [inline device-types/Supermicro/SYS-1019P-WTR.yaml:18]: Questionable as to whether this counts as a console port

**Review by DanSheps — CHANGES_REQUESTED** [review]

File: device-types/Supermicro/SYS-1019P-WTR.yaml (hunk 2 above)
Comment (DanSheps) [reply device-types/Supermicro/SYS-1019P-WTR.yaml:18]: Personally, I would remove USB interfaces all together.  Let the end-consumer decide if they want to model them.  I think there is a use case for having the USB ports, but I think it requires different handling as USB ports can be different things.

For example, on catalyst switches, there is a USB port on the back that can accept a 120G SSD, that would be best modelled as a inventory item labelled "USBFLASH1" or similar.

You can leave the VGA port as a console because it is a type of console but I suspect many people are not connecting into it.  This is the same as with the USB.  For example, lets say you want to model your KVM device, you might want those as a single "console-server-port" on the KVM but it spiders out and connects to both a monitor and USB port on the server.

TLDR;

Leave the VGA

Ditch the USB - The end user can model it how they desire

File: device-types/Supermicro/SYS-1019P-WTR.yaml (hunk 1 above)
Comment (vittore) [reply device-types/Supermicro/SYS-1019P-WTR.yaml]: Corrected in https://github.com/netbox-community/devicetype-library/commit/44d267b3c9d05a6b202660c02221d13d5549e9a0

Comment (vittore) [reply device-types/Supermicro/SYS-1019P-WTR.yaml]: Corrected in https://github.com/netbox-community/devicetype-library/commit/44d267b3c9d05a6b202660c02221d13d5549e9a0

File: device-types/Supermicro/SYS-1019P-WTR.yaml (hunk 2 above)
Comment (vittore) [reply device-types/Supermicro/SYS-1019P-WTR.yaml]: Corrected in https://github.com/netbox-community/devicetype-library/commit/44d267b3c9d05a6b202660c02221d13d5549e9a0

**Review by DanSheps — APPROVED** [review]

**Comment by DanSheps** [comment]
In the future, could you break out PR's like this into smaller PR's so that they are easier to review?

**Comment by vittore** [comment]
> In the future, could you break out PR's like this into smaller PR's so that they are easier to review?

Yes. It was my mistake.
Sorry.
